strictdoc/backend/sdoc/models/document.py

```diff
@@ -28,10 +30,9 @@ def __init__(
         self.title: str = title
         self.reserved_title: str = title
         self.config: DocumentConfig = (
-            config
-            if config is not None
-            else DocumentConfig.default_config(self)
+            config if config is not None else DocumentConfig.default_config(self)
```

Comment:
It does not look like you are changing anything here. This will probably go away when you run `invoke check`.